packages/web-components/fast-foundation/src/accordion/accordion.template.ts

```diff
@@ -14,8 +14,9 @@ export const accordionTemplate: (
 ) => ViewTemplate<Accordion> = (
     context: ElementDefinitionContext,
     definition: FoundationElementDefinition
-) => html`
+) => /* TODO: deprecate slot name `item` to only support default slot https://github.com/microsoft/fast/issues/XXXX */ html`
```

Comment:
@chrisdholt did you want to create the issue, or do you want me to, and you would add any additional details afterwards?

Reply:
Would you mind taking care of this? Thank you!

Reply:
Done!